lib/scss/components/cta-list/_cta-list.scss

```
@@ -0,0 +1,112 @@
+@charset "UTF-8";
+
+.su-cta-list-paragraph{
+
+    @include modular-spacing('padding-top', 3);
+    @include modular-spacing('padding-left', 2);
+    @include modular-spacing('padding-right', 2);
+
+    border-top: 1px solid $su-color-cool-grey-25;
+
+    @include grid-media-between('xs', 'md'){

```

`xs` == 0 
So this could be 
```suggestion
    @include grid-media-max('md'){

```